saleor/graphql/payment/types.py

```diff
@@ -52,6 +52,19 @@ class CreditCard(graphene.ObjectType):
     )
 
 
+class PaymentSource(graphene.ObjectType):
+    class Meta:
+        description = (
+            "Represents a payment source stored"
+            " for user in payment gateway, such as credit card."
```

Comment:
I believe it looks less weird/better if you put the trailing space in the above line instead at beginning. 

```python
x = (
    "hello "
    "world."
)